ultrasonic/src/main/res/values/strings.xml

```diff
@@ -332,6 +334,7 @@
     <string name="settings.wifi_required_title">Download on Wi-Fi only</string>
     <string name="song_details.all">%1$s%2$s</string>
     <string name="song_details.kbps">%d kbps</string>
+    <string name="song_details.nowplaying">%1$s&#160; • &#160;%2$d&#160; • &#160;%3$d kbps %4$s</string>
```

Comment:
I think that this is useless now, can be deleted.

Reply:
Ups, forget to delete that. Thanks for watching!